### DISCUSSION TITLE
Functionality to view subscriptions at Profile page

### DISCUSSION THREAD

**Discussion by swetachotalia** [Azure/api-management-developer-portal#2608] [Feature request]
Previously we had functionality to subscribe at Product level. With new UI, we are able to subscribe at Product and API level. But API level subscriptions are not visible on Profile page. Can you please add this feature as well?

**Comment by ghost** [comment]
@swetachotalia, thank you for opening this issue. We will triage it within the next few business days.

**Comment by harunrst** [comment]
"With the new UI", can you specify what are you referring to here? Because we do not support subscribing to an API through the developer portal, it is only allowed in Azure Portal. Subscribed products and APIs are already listed on the profile page. If it is not the case, please tell me how can I reproduce it.

**Comment by ghost** [comment]
@swetachotalia, thank you for requesting this feature.
